app/helpers/api/bapi/getCatalogueItems.js

```diff
@@ -0,0 +1,15 @@
+import { getData } from 'buying-catalogue-library';
+import { logger } from '../../../logger';
+import { solutionsApiUrl } from '../../../config';
+
+const getCatalogueItemsEndpoint = (supplierId, catalogueItemType) => (
+  `${solutionsApiUrl}/api/v1/catalogue-items?supplierId=${supplierId}&catalogueItemType=${catalogueItemType}`
```

Comment:
Is there ever a case where you don't need to send a supplier ID or a catalogue item type?

Reply:
I was thinking this and if the only other user of this endpoint will be associated services then the endpoint will be the same.

So I think we should leave this as is for now. Since if them values are not provided then it's not a valid endpoint for the FE